pkg/model/iam/iam_builder.go

```diff
@@ -284,6 +284,10 @@ func (r *NodeRoleMaster) BuildAWSPolicy(b *PolicyBuilder) (*Policy, error) {
 		addCalicoSrcDstCheckPermissions(p)
 	}
 
+	if b.Cluster.Spec.NodeTerminationHandler != nil && *b.Cluster.Spec.NodeTerminationHandler.Enabled && *b.Cluster.Spec.NodeTerminationHandler.EnableSqsTerminationDraining {
```

Comment:
This will hit some npr issues. You can use `fi.BoolValue(boolPointer)` to check these values.

Reply:
fixed